data/json/items/gun/50.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "m107a1",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Barrett M107A1",
+    "name_plural": "Barrett M107A1",
+    "description": "A large, shoulder-fired, .50 caliber anti-materiel rifle.  Its large size, recoil, and noise is offset by its damage and range.",
+    "weight": 12602,
+    "volume": 17,
+    "price": 650000, "//" : "Minimum list price for the bolt-action, Gun Digest, 2012.",
+    "bashing": 12,
+    "to_hit": -1,
+    "material": "steel",
+    "ammo": "50",
+    "ranged_damage": -5,
+    "range": 45,
+    "sight_dispersion": 60,
+    "recoil": 50,
+    "durability": 8,
+    "barrel_length" : 5,
+    "magazine_well": 1,
+    "magazines": [ [ "50", [ "m107a1mag" ] ] ],
+    "flags": "NEVER_JAMS"
+  },
+  {
+    "id": "m2browning",
```

Comment:
The same PR should migrate the sawn M2 browning. You need to make an entry in `migrations.json` then remove all other references to it